catroid/src/main/java/org/catrobat/catroid/ui/BaseExceptionHandler.java

```diff
@@ -30,6 +30,8 @@
 public class BaseExceptionHandler implements
 		java.lang.Thread.UncaughtExceptionHandler {
 
+	public static final String RECOVERED_FROM_CRASH = "RECOVERED_FROM_CRASH";
+
```

Comment:
To be consistent with other classes, the order of occurrences

1.  ```private static final long serialVersionUID = 1L;```
2. ```private static final String TAG = BaseExceptionHandler.class.getSimpleName();```
3. Anything else, e.g. ```public static final String RECOVERED_FROM_CRASH = "RECOVERED_FROM_CRASH";```

(1., 2. and 3. are each optional, no newline between 1. and 2., newline between 2. and 3.).

If files are very similar, it's easier for developer to work with any file and also they don't have to waste time on context switching.